CHANGELOG

```diff
@@ -0,0 +1,2 @@
+v 0.0.0
+  - Init
```

Comment:
I propose more descriptive information here. What is more maybe we can start with 0.1.0 version?